app/assets/stylesheets/collections/bulk-edit.scss

```diff
@@ -0,0 +1,51 @@
+.bulk-edit{
+  &-container {
+    display:flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: stretch;
+    height: calc(100vh - #{$nav-height} - #{$footer-height}); 
+    min-height: 200px;
+    @media only screen and (max-width: 850px) {
+      width: 90%;
+      height: calc(100vh - #{$nav-height} - #{$footer-height} - 15px); 
```

Comment:
These two `height` values exist on the parent container and are redundant. They're causing weird content overlaps on small screen sizes, too.

Reply:
Fixed